.github/workflows/docker-build-corretto-slim.yml

```diff
@@ -23,11 +22,10 @@ jobs:
 
       - run: sleep 5 # there's still a race condition for now
```

Comment:
Would be cool if you can followup checking if this is still needed, probably not.

Reply:
I pushed this to my fork which has OIDC set up and it logged in without a problem, except that I think the workflow failed because I don't have authority to push the tag as a last step: https://github.com/NathanielRN/aws-otel-java-instrumentation/actions/runs/1612693300

So I made PR #134 for this change.

Just curious, did we need to sleep because we can't run multiple `sts:AssumeRoleWithWebIdentity` too close to each other?